test/cql-pytest/README.md

```diff
@@ -202,3 +206,71 @@ and practices be followed when writing new tests:
    changing those tests without good reason, and don't add new tests to
    any file in it. Put new tests in the `cql-pytest` directory, in any
    place except the `cassandra_tests` subdirectory.
+
+# Installing Cassandra
+
+As explained above, the ability to run cql-pytest tests against Cassandra
+makes it easier to write correct tests, to ensure compatibility with
+Cassandra, and sometimes to write tests for new Cassandra-inspired features
+before developing the feature in Scylla (this is so-called "test-driven
+development"). Unfortunately, in recent years modern Linux distributions
+dropped their "cassandra" package, so to run Cassandra you'll need to
+install it manually, and this section explains how. It's very easy, and
+don't worry - you don't even need to learn how to run Cassandra, as the
+"test/cql-pytest/run-cassandra" tool will do it for you.
+
+To be able to run Cassandra, you'll need either Java 8 or 11 installed on
+your system - Cassandra does not support more recent versions of Java.
+However, this old Java only needs to be installed *alongside* your favorite
+version of Java - it does not need to be the default Java on your system.
+The "run-cassandra" script will automatically pick the right version of
+Java from multiple versions installed on your system. Of modern Fedora,
```

Comment:
Small typo; should say "_On_ modern Fedora" instead of "_Of_ modern Fedora"

Reply:
Done